docs/proposals/drafts/20211007-pluggable-authenticators.md

```diff
@@ -0,0 +1,146 @@
+---
+toc: true
+title: 2021-10 Pluggable Authenticators
+---
+
+## Introduce Pluggable Authenticators
+
+* **Owners**
+  * @tareqmamari
+
+## TL; DR
+
+We propose a pluggable authenticator approach to support plugging in custom authenticators in a self-registering manner.
+
+## Why
+
+This proposed approach allows the adopters of Observatorium to use their own custom authenticators' implementation other than the current OIDC, mTLS or Openshift authenticators.
```

Comment:
Nit: I think the number of authenticators an implementations should agree. I.e. either `authenticator's implementation` or `authenticators' implementations`.